six/modules/c++/six.sicd/source/ComplexXMLControl.cpp

```diff
@@ -117,6 +118,13 @@ ComplexXMLControl::getParser(const std::string& version) const
             // parser itself doesn't - so we can simply reuse the 1.0.1 parser.
             parser.reset(new ComplexXMLParser101(version, mLog));
         }
+        else if (minorVersion == "1" && patchVersion == "0")
+        {
+            // From a SIX standpoint, 1.2.0 is identical to 1.1.0. The changes
+            // get hanlded by XML_DATA_CONTENT.c. We just have a new version
```

Comment:
'hanlded' typo.  I'd actually reword the second sentence to "The only difference between 1.2 and 1.1 is some rules with how FTITLE can be populated, so we can again reuse the 1.0.1 parser"